readthedocs/search/models.py

```diff
@@ -32,6 +32,12 @@ class SearchQuery(TimeStampedModel):
         _('Query'),
         max_length=4092,
     )
+    total_results = models.IntegerField(
+        _('Total results'),
+        default=0,
+        # TODO: to avoid downtime, remove later.
+        null=True,
```

Comment:
If it's ok to not save some search queries during deploy I can just remove this.